cms/magento/README.md

```diff
@@ -0,0 +1,30 @@
+# Magento Deployer 7 Config
+
+```yaml
+import:
+    - 'vendor/unleashedtech/deployer-recipes/cms/magento/magento2.php'
```

Comment:
@dshumakerUT These two lines should be moved below line 14, given that this package now allows recipes to react based on earlier configuration.